### DISCUSSION TITLE
Ability to add non-standard JDBC params?

### DISCUSSION THREAD

**Discussion by tomwitt2** [rapiddweller/rapiddweller-benerator-ce#438] [Ideas]
I'd like to use the db-snapshot tool  to read the schema of a Snowflake database using JDBC.  This requires using the non-standard JDBC param of warehouse=<demo_xs> and the ability use SSO for login with the JDBC param of authentication=external_browser.  

Is there a way to specify these non-standard JDBC params when using the db-snapshot tools?

**Comment by PeterBrinkhoff** [comment]
Thanks @tomwitt2 for your input. We are checking this internally and will update you soon on this matter.
